pkg/webui/console/views/gateway-general-settings/basic-settings-form/index.js

```diff
@@ -201,6 +201,14 @@ const BasicSettingsForm = React.memo(props => {
         component={Input}
         autoComplete="on"
       />
+      <Form.Field
+        title={sharedMessages.packetBroker}
```

Comment:
```suggestion
        title={sharedMessages.packetBroker}
```

This should be more like "Peering via Packet Broker" and the default should be "Enabled" which is ticked.

Reply:
The gateway has a `disable_packet_broker_forwarding` field (makes things easier, since it will be `false` by default).

I am not sure how best to translate that into an "enabled" checkbox. Negating the checkbox feels like a not clean solution.

Reply:
Ok, knowing that this will be updated soon, (#3947) I'm fine merging it this way.